### PR TITLE
16 sp welcomemessage

### PR DESCRIPTION
**Pull request by potnurusrilatha**
please look in to this code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,19 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Home</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="stylesheet" href="./styles/general.css">
+    <link rel="stylesheet" href="./styles/variables.css">
+    <link rel="stylesheet" href="./styles/Fonts/KoHo_WOFF/KoHo-Bold.woff2">
```
Comment (TobiasKjernell) [inline index.html]: Here we can link the styleheet of Koho instead. so we get access to whole "font face library" automatically

File: styles/index.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+@font-face {
+    font-family: 'KoHo';
+    src: url('KoHo-Regular.woff2') format('woff2'),
+        url('KoHo-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+
```
Comment (TobiasKjernell) [inline styles/index.css:50]: we can take away this if we link stylesheet in html header. all @font face

File: index.html (hunk 1 above)
Comment (potnurusrilatha) [reply index.html]: ok

File: styles/index.css (hunk 2 above)
Comment (potnurusrilatha) [reply styles/index.css:50]: ok

**Review by jasminedns — APPROVED** [review]

**Review by Moon2024-light — APPROVED** [review]
good

**Pull request merged**